tools/archive/templates/archive/issue.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Issue {{number}}</title>
+    <link href="https://webcompat.com/dist/webcompat.css" type="text/css" rel="stylesheet">
+    <style>table, td, th { border: 1px black solid; border-collapse: collapse;}</style>
```

Comment:
I needed to remind myself if our CSP would allow this, https://github.com/webcompat/webcompat.com/blob/9b39a457bab1453ed9015a8cd5033c1e6837818f/webcompat/helpers.py#L546

And it does!